src/components/footer.stories.tsx

```diff
@@ -0,0 +1,25 @@
+
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { Footer } from "@guardian/src-footer"
+
+interface FooterProps {
+    showBackToTop: boolean;
```

Comment:
I don't think these are used?

Reply:
updated stories to be defaulted as true will add a switch between true and false to render ccpa content